project/LinkerdBuild.scala

```diff
@@ -127,7 +133,7 @@ object LinkerdBuild extends Base {
       .withTests()
 
     val k8s = projectDir("namer/k8s")
-      .dependsOn(LinkerdBuild.k8s, core)
+      .dependsOn(LinkerdBuild.k8s, LinkerdBuild.istio, core)
```

Comment:
I think rather than making the `namer/k8s` project depend on `istio`, it might make more sense to move the Istio namers to a new `namer/istio` project?

Reply:
Oh yeah, great point. I didn't want to change too much, hence the k8s->istio dep, but this looks like an easy one and maybe we'll be able to break it completely in one go.